src/nvim/screen.c

```diff
@@ -1925,10 +1925,10 @@ static int line_putchar(LineState *s, schar_T *dest, int maxcells, bool rl)
 /// @return number of chars added to \param p
 static size_t
 fill_foldcolumn(
-    char_u *p,
-    win_T *wp,
-    foldinfo_T foldinfo,
-    linenr_T lnum
+                char_u *p,
+                win_T *wp,
+                foldinfo_T foldinfo,
+                linenr_T lnum
```

Comment:
Is aligning function parameters with the opening paren of a function definition already part of the Nvim style guide? If not, I'd like to throw my opinion into the ring to not do this and just have the parameter list indented twice (as it is now) to minimize diff noise any time a function name changes. I personally also find it less readable, particularly when you have a long function name and the parameter list is halfway across the screen.

Reply:
> Is aligning function parameters with the opening paren of a function definition already part of the Nvim style guide?

[Yes it is](https://neovim.io/develop/style-guide.xml?showone=Function_Declarations_and_Definitions#Function_Declarations_and_Definitions). That being said, if there ever was good opportunity to revise the style rules it would be now :) . I agree it looks kinda ridiculous for functions with long names or functions with many arguments.

Reply:
It is also part of our style to _not_ have a linebreak per parameter no reason, but fill with parameters until the column limit is reached, and _then_ add a line break aligned to the start param (there are legitimate exceptions to this, but this function is not. it is just vim code which hasn't been adjusted yet).

Reply:
> I agree it looks kinda ridiculous for functions with long names or functions with many arguments.

The style guide even mentions this case explicitly already.